submissions/yaripey/dom-task/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/main.css">
+  <script src="js/main.js"></script>
+</head>
+
+
+<body>
+
+  <header>
+    <a href="#" class="toggle"></a>
+    <img class="logo" src="img/logo.png" alt="logo">
+  </header>
+
+  <div class="middle">
+
+    <aside>
+      <nav class="main-navigation">
+        <ul>
+          <li><button class="focused" id="about">About</button></li>
+          <li><button id="finn">Finn the Human</button></li>
+          <li><button id="jake">Jake the Dog</button></li>
+          <li><button id="marcy">Marceline</button></li>
+          <li><button id="pb">Princess Bubblegum</button></li>
+        </ul>
+      </nav>
+    </aside>
+
+    <main>
+      Adventure Time is an American fantasy animated television series created by Pendleton Ward for Cartoon Network.
```

Comment:
Why you need this, if you generate content by js?

Reply:
Removed it, thanks.